src/shared/config.py

```diff
@@ -55,6 +55,8 @@ class Config(BaseModel):
     processing: ProcessingConfig
     server: Optional[str] = None
     server_port: int = 5001
+    enable_background_scheduler: bool = False
```

Comment:
nit, you could probably combine these into on 

`background_update_interval_minute: Optional[int]`

Reply:
Ah yeah, probably could. That seems cleaner than needing a true/false and an interval since the interval is only needed if you are running the scheduler.